innovation.game.php

```diff
@@ -1711,6 +1802,12 @@ function notifyWithTwoPlayersInvolved($card, $transferInfo, $progressInfo) {
                 $message_for_opponent = clienttranslate('${player_name} transfers ${<}${age}${>} ${<<}${name}${>>} from his score pile to ${your} score pile.');
                 $message_for_others = clienttranslate('${player_name} transfers a ${<}${age}${>} from his score pile to ${opponent_name}\'s score pile.');
                 break;
+
+            case 'score->achievements':
+                $message_for_player = clienttranslate('${You} transfer ${<}${age}${>} ${<<}${name}${>>} from your score pile to ${opponent_name}\'s achievements.');
+                $message_for_opponent = clienttranslate('${player_name} transfers ${<}${age}${>} ${<<}${name}${>>} from his score pile to ${your} achievements.');
```

Comment:
{player_name} transfers a ${<}${age}${>} from his score pile to ${your} achievements.